webpack/webpack5-loader-load-themed-styles/tsconfig.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "./node_modules/@rushstack/heft-node-rig/profiles/default/tsconfig-base.json",
+  "compilerOptions": {
+    "types": ["heft-jest", "node"]
+  },
+  "include": ["src", "loader-utils.d.ts"]
```

Comment:
Where is `loader-utils.d.ts`?

Reply:
It is gone now since it appears that loader-utils is not even needed anymore given the new `loaderContext` api to stringify requests.